src/main/CommandHandler.cpp

```diff
@@ -661,7 +661,8 @@ CommandHandler::dumpProposedSettings(std::string const& params,
             return;
         }
 
-        retStr = xdr_to_string(ptr->toXDR(), "ConfigUpgradeSet");
+        retStr =
```

Comment:
Why the rename? This may be a little pedantic, but isn't `ConfigSettingsEntries` a subset of `ConfigUpgradeSet`? For example, a protocol upgrade would still be a `ConfigUpgradeSet`, so dumping it as `ConfigSettingsEntries` would be incorrect.

Reply:
So this isn't just a rename. This line was changed to return the `xdr::xvector<stellar::ConfigSettingEntry>` inside `ConfigUpgradeSet` to match what is now returned in the detailed `sorobaninfo` endpoint.